examples/datadogagent/v2alpha1/datadog-agent-openshift.yaml

```diff
@@ -18,13 +18,19 @@ spec:
       apiKey: <DATADOG_API_KEY>
       appKey: <DATADOG_API_KEY>
     criSocketPath: /var/run/crio/crio.sock
+    kubelet:
+      tlsVerify: false
```

Comment:
is it mandatory?

Reply:
I think another solution is to mount the openshift root certificate

Reply:
can we add a comment to say that `tlsVerify: false` is needed of the kubelet certificate is a self-signed certificat. 

and another way is to mount the CA certificat that was used to create the kubelet certificate.